Alcolator/Alcolator/ViewController.m

```diff
@@ -47,35 +49,25 @@ - (IBAction)buttonPressed:(UIButton *)sender {
     
     //calculate the amount of alcohol in the beer
     int numberOfBeers = self.beerCountSlider.value;
-    int ouncesInOneBeerGlass = 12; //assumed 12oz bottles
-    float alcoholPercentageOfBeer = [self.beerPercentTextField.text floatValue] / 100;
-    float ouncesOfAlcoholPerBeer = ouncesInOneBeerGlass * alcoholPercentageOfBeer;
-    float ouncesOfAlcoholTotal = numberOfBeers * ouncesOfAlcoholPerBeer;
-
-    //now calculate the equivalent of wine . . .
-    float ouncesOfWineInOneGlass = 5; //assumed 5oz glass
-    float alcoholPercentageOfWine = 0.13; //assumed average
-    float ouncesOfAlcoholPerGlassOfWine = ouncesOfWineInOneGlass * alcoholPercentageOfWine;
-    float numberOfWineGlassesEquivalentToNumberOfBeers = ouncesOfAlcoholTotal / ouncesOfAlcoholPerGlassOfWine;
     
-    //decide whether to use beers or beer, glass or glasses
-    NSString *beerText;
-    if ( numberOfBeers == 1 ) {
-        beerText = NSLocalizedString(@"beer", @"singular beer");
-    } else {
-        beerText = NSLocalizedString(@"beers", @"plural of beer");
-    }
+    Beer *myBeer = [[Beer alloc] init];
```

Comment:
I might create a property on the view controllers... for example, on this `ViewController` I would create something like this:
```
@property (strong, nonatomic) Beer* beer;
@property (strong, nonatomic) Wine* wine;
```

And then, in `- (void)viewDidLoad` I would initialize them... in `WhiskeyViewController` it's kind of handy because you only need to initialize and declare a `whiskey` property.

We'll need to update the alcohol percentage of the beer if that text field ever changes, but this might be best (because right now, with this implementation, we instantiate a new `Beer` object and `Wine` or `Whiskey` object every time someone press the calculate button.